Tools/cmake/FindDocGenerator.cmake.in

```diff
@@ -0,0 +1,17 @@
+find_package(JsonGenerator REQUIRED)
+
+function(DocGenerator)
+    set(oneValueArgs INPUT)
+
+    cmake_parse_arguments(Argument "${oneValueArgs}" ${ARGN} )
+
+    if(Argument_INPUT)
+        set(PLUGIN_NAME ${ARGN})
```

Comment:
Based on onevalueargs setting I suppose this function will be invoked like this DocGenerator(INPUT pluginName) so this won't work either as ARGN will be "INPUT pluginName".